<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchSettings</class>
 <widget class="QWidget" name="SearchSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layoutParent">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>750</width>
        <height>689</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutScrollArea">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelMatcher">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Matchers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkExact">
         <property name="toolTip">
          <string>Matches text exactly as it appears in subtitles</string>
         </property>
         <property name="text">
          <string>Exact Matcher</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkDeconj">
         <property name="toolTip">
          <string>Matches vocabulary by deconjugating using known rules.
Includes deconjugation information in results.</string>
         </property>
         <property name="text">
          <string>Deconjugation Matcher</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkMecabIpadic">
         <property name="toolTip">
          <string>Matches vocabulary using MeCab with IPAdic</string>
         </property>
         <property name="text">
          <string>MeCab IPAdic Matcher</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLimit">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Result Limit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinLimitResults">
         <property name="toolTip">
          <string>Sets the maximum number of results that can show up in a search.
Larger values decrease performance.
Zero is equivalent to no limit.</string>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMethod">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Search Method</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxMethod">
         <property name="toolTip">
          <string>Sets how searches are started.&lt;br&gt;
&lt;b&gt;Hover:&lt;/b&gt; Searches start when the mouse hovers over a subtitle.&lt;br&gt;
&lt;b&gt;Modifier:&lt;/b&gt; Searches start when the modifier key is held.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameHover">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelDelay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Search Delay</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxDelay">
            <property name="toolTip">
             <string>Sets the delay between hovering over text and executing a search.
Changing this value from the default is not recommended.</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDelayMS">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelModifier">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Search Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxModifier">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The modifier key that will be used to trigger searches.&lt;/p&gt;&lt;p&gt;This applies to subtitle searches if the search method is set to &lt;span style=&quot; font-weight:600;&quot;&gt;Modifier&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutCheckHide">
         <item>
          <widget class="QCheckBox" name="checkMiddleMouse">
           <property name="text">
            <string>Search using middle mouse button</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkHideSubs">
           <property name="text">
            <string>Hide mpv subtitles when subtitle search is visible</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkHideSearch">
           <property name="text">
            <string>Hide subtitle search when playing media</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkHoverPause">
           <property name="toolTip">
            <string>Pauses the player when the mouse is moved over the subtitle search</string>
           </property>
           <property name="text">
            <string>Pause on subtitle search hover</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkAutoPlayAudio">
           <property name="toolTip">
            <string>Automatically play audio for the top search result</string>
           </property>
           <property name="text">
            <string>Auto-play search result audio</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutReplace">
         <item>
          <widget class="QCheckBox" name="checkReplaceNewLines">
           <property name="text">
            <string>Replace new lines in subtitles with</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditReplace"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutGlossaryStyle">
         <item>
          <widget class="QLabel" name="labelGlossaryStyle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Separate glossary entries with</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboGlossaryStyle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Bullet Points</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Line Breaks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vertical Bars</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelRemoveRegex">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Remove Regex in Subtitles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineRemoveRegex">
         <property name="toolTip">
          <string>Removes all pieces of text in the primary subtitle that match the given regular expression.
Unlike &quot;Replace new line with&quot;, this modifies subtitles globally.
Changes to the primary subtitle will appear in the subtitle list, Anki cards, and subtitle search.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Reset|QDialogButtonBox::StandardButton::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
